packages/firestore/lite/src/api/database.ts

```diff
@@ -300,6 +301,38 @@ export function getFirestore(app: FirebaseApp): FirebaseFirestore {
   ).getImmediate() as FirebaseFirestore;
 }
 
+/**
+ * Modify this instance to communicate with the Cloud Firestore emulator.
+ *
+ * Note: This must be called before this instance has been used to do any
+ * operations.
+ *
+ * @param firestore - The Firestore instance to configure to connect to the
+ * emulator.
+ * @param host - the emulator host (ex: localhost).
+ * @param port - the emulator port (ex: 9000).
+ */
+export function useFirestoreEmulator(
+  firestore: FirebaseFirestore,
+  host: string,
+  port: number
+): void {
+  firestore = cast(firestore, FirebaseFirestore);
+
+  if (firestore._getSettings().host !== DEFAULT_HOST) {
```

Comment:
Out of scope here but I recently realized that this warning shows up if you repeatedly call `useEmulator()` with the same arguments.  If you feel like fixing it, we could check `settings.host !== DEFAULT_HOST && settings.host !== host` or something.

Reply:
Let me fix that.